test/python/test_authn_api.py

```diff
@@ -377,6 +394,26 @@ def test_authenticate_service_404(self):
 
         self.assertEqual(context.exception.status, 404)
 
+    def test_oidc_authenticate_200(self):
+        """Test case for oidc_authenticate 200 response"""
+        api_config.setup_oidc_webservice()
+        id_token = get_oidc_id_token()
+
+        _, status, _ = self.api.oidc_authenticate_with_http_info(
+            'test',
+            self.account,
+            id_token=id_token
+        )
+        self.assertEqual(status, 200)
+
+    def test_oidc_authenticate_401(self):
```

Comment:
I think including a test for a `502` response would be nice. Not sure how much control we would have over receiving this sort of status code but I feel like it would be good to verify.

Reply:
Unfortunately the 502 response requires that there is a bad TLS setup between Conjur and the OIDC server so its not really feasible to test it without entirely restarting the Conjur server